shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -70,6 +72,36 @@ - (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent*)event {
   return NO;
 }
 
+- (void)applyBackdropFilter {
+  if (_backdropFilterApplied)  // TODO EMILY: This check is not needed, but it might improve
```

Comment:
Instead of bool, store the gaussianFilter instance and compare the inputRadius with the new blur intensentiy we got from the DLImageFilter object.